doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -155,3 +158,40 @@ You can query the ``refcats`` collection to see whether your htm shards appear:
     butler query-datasets --collections refcats REPO
 
 For LSST staff using ``lsst-devl``, see the `Reference catalogs policy <https://developer.lsst.io/services/datasets.html#reference-catalogs>`_ in the Developer Guide for additional policy about adding reference catalogs to the common repo.
+
+.. _lsst.meas.algorithms-refcat-existing:
+
+5. What to do with existing reference catalogs?
+===============================================
+
+Already existing reference catalogs (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers) can also be ingested into the butler, negating the need to convert them to the LSST format yourself.
+To ingest an already existing converted reference catalog, first create a suitable filename to htm7-index astropy lookup table, and then follow the steps above to `ingest the files into the butler <lsst.meas.algorithms-refcat-ingest>`.
+
+This is an example script that constructs a conversion lookup table for all converted FITS files in the `refcat_dir` directory, outputting to the `out_dir` directory:
+
+.. code-block:: python
+
+    import os
+    import glob
+    import astropy.table
+
+    refcat_dir = "/path/to/my/refcat/directory"
+    out_dir = "/path/to/my/output/directory"
+
+    out_file = f"{out_dir}/{os.path.basename(refcat_dir)}.ecsv"
+    print(f"Saving to: {out_file}")
+
+    table = astropy.table.Table(names=("filename", "htm7"), dtype=("str", "int"))
+    files = glob.glob(f"{refcat_dir}/[0-9]*.fits")
+
+    for i, file in enumerate(files):
+        # running status, overwriting each print statement as it proceeds
+        print(f"{i}/{len(files)} ({100*i/len(files):0.1f}%)", end="\r")
+
+        # extract file index; add row to table
+        file_index = int(os.path.basename(os.path.splitext(file)[0]))
+        table.add_row((file, file_index))
+
+    table.write(out_file)
+
+Once this script is complete, finalize the reference catalog ingestion by following the `file ingestion instructions above <lsst.meas.algorithms-refcat-ingest>`.
```

Comment:
"finalize"->"finish", maybe?

And the final link doesn't work in the output HTML. I think you need a `:ref:` before these.

https://developer.lsst.io/restructuredtext/style.html#linking